e2e/react-start/hmr/src/hmr/server-fn-hmr-factory.ts

```
@@ -0,0 +1,4 @@
+import { createClientOnlyFn, createServerOnlyFn } from '@tanstack/react-start'
```

ESLint

[error] 'createClientOnlyFn' is defined but never used.

(unused-imports/no-unused-imports)